src/js/components/locations-configuration/AddZoneModal.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+
+import PropTypes from 'prop-types';
+import { getTranslate } from 'react-localize-redux';
+import { connect } from 'react-redux';
+import Alert from 'react-s-alert';
+
+import { hideSpinner, showSpinner } from 'actions';
+import ModalWrapper from 'components/form-elements/ModalWrapper';
+import apiClient from 'utils/apiClient';
+import Translate, { translateWithDefaultMessage } from 'utils/Translate';
+
+
+class AddZoneModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      values: {
+        active: true,
+      },
+    };
+  }
+
+  handleSubmit(values) {
+    this.props.showSpinner();
+    apiClient.post('/openboxes/api/locations/', {
+      ...values,
+      'parentLocation.id': `${this.props.locationId}`,
+      'locationType.id': `${this.props.zoneTypeId}`,
```

Comment:
As per discussion during standup: it might be safer to have a select since there might be multiple zone types in the system. You can make it preselected if there is only one location type with `ZONE` code in the options.